docs/guide/field_type_reference.md

```diff
@@ -1450,7 +1540,7 @@ This Field Type is used as fallback and for testing purposes.
 
 ### Description
 
-The Null Field Type serves as an aid when migrating from eZ Publish Platform and earlier versions. It is a dummy for legacy Field Types that are not implemented in eZ Platform.
+The Null Field Type serves as an aid when migrating from eZ Publish and earlier versions. It is a dummy for legacy Field Types that are not implemented in eZ Platform.
```

Comment:
Actually "Publish Platform" can stay here, it's another name for 5.x (4.x was "Publish" and 5.x was "Publish Platform")